tsl/test/sql/CMakeLists.txt

```diff
@@ -103,6 +97,22 @@ if((${PG_VERSION_MAJOR} GREATER_EQUAL "14"))
   endif()
 endif()
 
+# Run cagg migration tests only on PG version < 15
+if((${PG_VERSION_MAJOR} LESS "15"))
+  if(CMAKE_BUILD_TYPE MATCHES Debug)
+    list(
+      APPEND
+      TEST_FILES
+      cagg_migrate_integer.sql
+      cagg_migrate_integer_dist_ht.sql
+      cagg_migrate_timestamp.sql
+      cagg_migrate_timestamp_dist_ht.sql
+      continuous_aggs_deprecated.sql)
+  else()
+    list(APPEND TEST_FILES cagg_errors_deprecated.sql)
+  endif()
+endif()
```

Comment:
I guess we need to think more about it... we should be able to execute the regression tests in PG15 for cagg migrations because we'll continue supporting it ...

Reply:
Ok, agreed.
What i did is restrict caggs with old formats only on release builds with PG15. This will let us run above tests on debug version till we make those tests compatible with PG15.